api/users/serializers.py

```diff
@@ -1,6 +1,30 @@
 from rest_framework import serializers as ser
-
 from api.base.serializers import JSONAPISerializer, LinksField, Link
+from website.models import User
+
+
+class ProfileDictSerializer(ser.ListField):
+    start_year = ser.CharField(allow_blank=True, allow_null=True)
+    start_month = ser.IntegerField(allow_null=True)
```

Comment:
Is this how we want to represent start and end dates in v2? I wonder if it'd more user-friendly to use iso-formatted dates. Best to do some research on best practices here.

ping @brianjgeiger for feedback.